docs/integrations/builtin/credentials/cal.md

```diff
@@ -14,15 +14,21 @@ You can use these credentials to authenticate the following nodes with Cal.
 
 Create a [Cal](https://www.cal.com/) pro account.
 
-## Using API keys
+## Supported authentication methods
 
-Your API requests are authenticated using API keys. Any request that doesn't include an API key will return an error.
+- API key
 
-1. Log into your Cal account.
-2. Click on **Settings**.
-3. There you will find the API keys section which allows you to generate a key for use. More information on this can be found on the [Cal Authentication](https://developer.cal.com/api/authentication){:target=_blank} page.
-4. Use API key in your Cal trigger node credentials in n8n.
+## Related resources
 
+Refer to [Cal's API documentation](https://cal.com/docs/enterprise-features/api#api-server-specifications){:target=_blank .external-link} for more information about the service.
 
-![Getting Cal credentials](/_images/integrations/builtin/credentials/cal/getting-api-key.gif)
+## Using API key
+
+To configure this credential, you'll need:
+
+- An **API Key**: Refer to the [Cal API Quick Start documentation](https://cal.com/docs/enterprise-features/api/quick-start){:target=_blank .external-link} for information on how to generate a new API key.
+
+/// note | Hosted API
+Cal previously supported [a hosted API](https://cal.com/docs/enterprise-features/api/quick-start/hosted-api-through-cal-com){:target=_blank .external-link}, but they have since discontinued that service. You do not need to make any edits to the **Host** field within the credential.
```

Comment:
```suggestion
Cal previously supported [a hosted API](https://cal.com/docs/enterprise-features/api/quick-start/hosted-api-through-cal-com){:target=_blank .external-link}, but they have since discontinued that service. You don't need to make any edits to the **Host** field within the credential.
```